ide/texmacspp.ml

```diff
@@ -520,6 +520,15 @@ let rec tmpp ?loc v =
       xmlScope ?loc "delimit" name ~attr:["delimiter",tag] []
   | VernacDelimiters (name,None) ->
       xmlScope ?loc "undelimit" name ~attr:[] []
+  | VernacNotationActivation (Some name,_,true) ->
+      xmlScope ?loc "activate" name ~attr:[] []
+  | VernacNotationActivation (Some name,_,false) ->
+      xmlScope ?loc "deactivate" name ~attr:[] []
+  | VernacNotationActivation (None,Some name,true) ->
+      xmlScope ?loc "activate" name ~attr:[] []
+  | VernacNotationActivation (None,Some name,false) ->
+      xmlScope ?loc "deactivate" name ~attr:[] []
+  | VernacNotationActivation (None,None,_) -> assert false
```

Comment:
Using the table system in `library/goptions` could save some effort here, and additionally make the life for IDEs wanting to introspect this table much easier.